jmh/src/main/java/org/roaringbitmap/iteration/IteratorsBenchmark32.java

```diff
@@ -24,6 +21,42 @@
 @OutputTimeUnit(TimeUnit.MILLISECONDS)
 public class IteratorsBenchmark32 {
 
+
+  @Benchmark
+  public int testBatch_a(BenchmarkState benchmarkState) {
+    BatchIterator it = benchmarkState.bitmap_a.getBatchIterator();
+    int[] buffer = benchmarkState.buffer;
+    int batch = 0;
+    while (it.hasNext()) {
+      batch = it.nextBatch(buffer);
+    }
+    return buffer[batch];
+  }
+
+
+  @Benchmark
+  public int testBatch_b(BenchmarkState benchmarkState) {
+    BatchIterator it = benchmarkState.bitmap_b.getBatchIterator();
+    int[] buffer = benchmarkState.buffer;
+    int batch = 0;
+    while (it.hasNext()) {
+      batch = it.nextBatch(buffer);
```

Comment:
Here I would iterate over the buffer otherwise I think that the test is not fair against standard iterators.

Reply:
Agreed.

Reply:
I doubt this can actually be measured unless each value is folded into an aggregate, I would expect an optimising compiler to realise that only the last element is used. I will delete these new benchmarks and start a new set of comparative benchmarks.